maven-failsafe-plugin/pom.xml

```diff
@@ -108,7 +108,7 @@
                     <dependency>
                         <groupId>org.apache.maven.surefire</groupId>
                         <artifactId>surefire-shadefire</artifactId>
-                        <version>3.0.0-M6</version> <!-- ${shadedVersion}, but resolved due to https://issues.apache.org/jira/browse/MRELEASE-799 -->
+                        <version>3.0.0-M7</version> <!-- ${shadedVersion}, but resolved due to https://issues.apache.org/jira/browse/MRELEASE-799 -->
```

Comment:
maybe we could have a property such `<surefire.plugin.version.runtime>` or another name to avoid such level of duplication?

Reply:
Pleas look at https://issues.apache.org/jira/browse/MRELEASE-799

Reply:
🤦😩